src/init.c

```diff
@@ -170,6 +171,7 @@ R_CallMethodDef callMethods[] = {
 {"CinitLastUpdated", (DL_FUNC) &initLastUpdated, -1},
 {"Ccj", (DL_FUNC) &cj, -1},  
 {"Ccoalesce", (DL_FUNC) &coalesce, -1},
+{"CfifelseR", (DL_FUNC) &fifelseR, 3},
```

Comment:
Huh, TIL what that third argument is for (number of arguments)...

@mattdowle is there any reason we don't specify the number of arguments for any other routines?

> For routines with a variable number of arguments..., one specifies -1 for the number of arguments which tells R not to check the actual number passed.

Maybe an efficiency thing (# args is not checked)?